Tasks.php

```diff
@@ -34,4 +38,42 @@ public function regenerateReleasedContainers()
             $tagManager->regenerateReleasedContainers();
         }
     }
+
+    public function deleteContainersForNonExistingSite()
+    {
+        /** @var TagManager $tagManager */
+        $tagManager = $this->pluginManager->getLoadedPlugin('TagManager');
+        if (!$tagManager) {
+            return;
+        }
+        $containerDao = new ContainersDao();
+        $containers = $containerDao->getAllContainers();
+        $siteIdsDeleted = [];
+        foreach ($containers as $container) {
+            $idsite = $container['idsite'];
+            if ($idsite === 286) {
+                $a=1;
+            }
```

Comment:
Is this left over from testing? It doesn't appear to do anything.

```suggestion
```

Reply:
@snake14 Sorry I was testing and pushed this